components/apimgt/org.wso2.carbon.apimgt.impl/src/test/java/org/wso2/carbon/apimgt/impl/APIManagerConfigurationTest.java

```diff
@@ -53,4 +55,76 @@ public void testPublisherPropertiesUndefined() throws XMLStreamException {
         Assert.assertTrue(extractedProperties.isEmpty());
 
     }
+
+    @Test
+    public void testEnvironmentsConfigProvided() throws XMLStreamException, APIManagementException {
```

Comment:
is there any mandatory parameters for this config ? Did we write tests to validate those ?